frontend/src/metabase/actions/components/ActionViz/ActionDashcardSettings.unit.spec.tsx

```diff
@@ -456,6 +456,16 @@ describe("ActionViz > ActionDashcardSettings", () => {
     ).toBeInTheDocument();
   });
 
+  it("should be valid and not crash when the action does not have parameters", async () => {
+    const { closeSpy } = setup({
+      dashcard: createMockActionDashboardCard({
+        action: createMockQueryAction(),
+      }),
+    });
+    await userEvent.click(screen.getByRole("button", { name: "Done" }));
```

Comment:
The test checks that for the case where `parameters` are `undefined` there is no bug in `isFormInvalid` logic, i.e. the settings form is considered valid.